.gitattributes

```diff
@@ -0,0 +1 @@
+*.js text eol=lf
```

Comment:
This makes line endings sane on Windows (so it's not trying to convert them into CRLF): https://github.com/Microsoft/vscode/issues/2957